diff --git a/dist/auth.d.ts b/dist/auth.d.ts
index 98c409ef3c5eb557a8e8bcd344e0cf33732d4fdc..f069201f3840819edb67c50b6a71bec083578444 100644
--- a/dist/auth.d.ts
+++ b/dist/auth.d.ts
@@ -32,8 +32,8 @@ export declare class Auth {
     getRedirectUrl(host: string, redirectUrl?: string): Promise<string>;
     handleProviderCallback(event: RequestEvent, provider: Provider): Promise<EndpointOutput>;
     handleEndpoint(event: RequestEvent): Promise<EndpointOutput>;
-    get: RequestHandler;
-    post: RequestHandler;
+    GET: RequestHandler;
+    POST: RequestHandler;
     getSession: GetSession;
 }
 export {};
diff --git a/dist/auth.js b/dist/auth.js
index 49a673a0b4e0db5b2f786c986f90bf23da61d292..9ce9780a1433a3b66760ce3593bded283cb98807 100644
--- a/dist/auth.js
+++ b/dist/auth.js
@@ -31,7 +31,7 @@ var jsonwebtoken__namespace = /*#__PURE__*/_interopNamespace(jsonwebtoken);
 
 class Auth {
   constructor(config) {
-    this.get = async (event) => {
+    this.GET = async (event) => {
       const { url } = event;
       if (url.pathname === this.getPath("csrf")) {
         return { body: "1234" };
@@ -45,7 +45,7 @@ class Auth {
       }
       return await this.handleEndpoint(event);
     };
-    this.post = async (event) => {
+    this.POST = async (event) => {
       return await this.handleEndpoint(event);
     };
     this.getSession = async (event) => {